spyder/plugins/ipythonconsole/plugin.py

```diff
@@ -28,10 +28,11 @@
 from qtconsole.manager import QtKernelManager
 from qtpy.compat import getopenfilename
 from qtpy.QtCore import Qt, Signal, Slot
-from qtpy.QtWidgets import (QApplication, QCheckBox, QDialog, QDialogButtonBox,
-                            QFormLayout, QGridLayout, QGroupBox, QHBoxLayout,
-                            QLabel, QLineEdit, QMessageBox, QPushButton,
-                            QTabWidget, QVBoxLayout, QWidget)
+from qtpy.QtWidgets import (QApplication, QButtonGroup, QCheckBox, QDialog,
+                            QDialogButtonBox, QFormLayout, QGridLayout,
+                            QGroupBox, QHBoxLayout, QLabel, QLineEdit,
+                            QMessageBox, QPushButton, QRadioButton,
+                            QSpacerItem, QTabWidget, QVBoxLayout, QWidget)

```

Comment:
I think most of these imports are not needed now that you moved the dialog to a different file. So please check if they are still used here or not.

Reply:
@bcolsen, I think these imports (in the plugin, I mean) also need some cleanup.

Reply:
Oops. I need to read more closely. I'll get those too.